libs/cardano-ledger-core/src/Cardano/Ledger/Slot.hs

```diff
@@ -74,3 +74,6 @@ epochInfoSize ::
   EpochNo ->
   ShelleyBase EpochSize
 epochInfoSize ei = lift . EI.epochInfoSize ei
+
+-- ================================================
+-- TreeDiff ToExpr instances
```

Comment:
But no instances defined?

Reply:
Instances have been moved to Cardano.Ledger.TreeDiff, where the ToExpr class is reexported, so any one who knows about TreeDiff sees these instances. So this should be deleted.